### PR TITLE
Fix CI pipeline in GPU machines

### PR DESCRIPTION
**Pull request by mdiazmel**
Add path to locate pipx in GPU machine

### CODE REVIEW

File: .jenkins/Jenkinsfile [hunk 1]
```diff
@@ -259,8 +259,9 @@ pipeline {
             environment {
               CONDA_HOME = "$HOME/miniconda3"
               CONDA_ENV = "$WORKSPACE/env"
+              PATH = "$HOME/.local/bin:$PATH"
               TMP_DIR = "$HOME/tmp"
-              INPUT_DATA_DIR = "/mnt/data/clinicadl_data_ci/data_ci"
+              INPUT_DATA_DIR = '/mnt/data/clinicadl_data_ci/data_ci'
```
Comment (NicolasGensollen) [inline .jenkins/Jenkinsfile:264]: Not super important, but I don't see why these were changed from " to '.

**Review by NicolasGensollen — APPROVED** [review]
LGTM @mdiazmel !

File: .jenkins/Jenkinsfile (hunk 1 above)
Comment (mdiazmel) [reply .jenkins/Jenkinsfile:264]: Thanks for noticing this @NicolasGensollen, I hadn't realized that!
After investigating, it seems that the groovy linter in my editor (vscode) doesn't like the double " for this line (and it changes automatically). This [link](https://stackoverflow.com/questions/6761498/whats-the-difference-of-strings-within-single-or-double-quotes-in-groovy) gives an explanation. But I have to  confess that it is a bit disturbing to mixing double " and single ' .

Comment (NicolasGensollen) [reply .jenkins/Jenkinsfile:264]: Thanks for the link @mdiazmel ! 👍

**Pull request merged**